satellite/accounting/live/live_test.go

```diff
@@ -14,35 +15,190 @@ import (
 	"go.uber.org/zap/zaptest"
 	"golang.org/x/sync/errgroup"
 
+	"storj.io/storj/internal/testcontext"
 	"storj.io/storj/internal/testrand"
+	"storj.io/storj/satellite/accounting"
+	"storj.io/storj/storage/redis/redisserver"
 )
 
 func TestPlainMemoryLiveAccounting(t *testing.T) {
```

Comment:
nit: you can write one testsuite that tests both implementations rather than writing tests for each